build_macos

```diff
@@ -1,14 +1,23 @@
 #!/usr/bin/env bash
 
+DEP="
+-I lib/gl3w/include lib/gl3w/src/gl3w.c \
+-I lib/glfw/include -L lib/glfw/src -framework Cocoa -framework IOKit -framework CoreVideo -lglfw3 \
```

Comment:
Try and keep similar dependencies on one line. So all the glfw stuff on line 5, all the mac items on the next.